lib/roo_on_rails/checks/sidekiq/sidekiq.rb

```diff
@@ -0,0 +1,48 @@
+require 'roo_on_rails/checks/base'
+require 'thor'
+
+module RooOnRails
+  module Checks
+    module Sidekiq
+      class Sidekiq < Base
+        WORKER_PROCFILE_LINE = "worker: RAILS_MAX_THREADS=$SIDEKIQ_DB_POOL_SIZE \
+                                DB_REAPING_FREQUENCY=$SIDEKIQ_DB_REAPING_FREQUENCY \
```

Comment:
Ah, no, he's generating the Sidekiq configuration. In which case yes, it should be.